<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GpReport</class>
 <widget class="QWidget" name="GpReport">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>912</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" rowspan="4">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0" colspan="8">
    <widget class="QTableWidget" name="twReport"/>
   </item>
   <item row="0" column="7" rowspan="4">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="2">
    <widget class="QDateTimeEdit" name="sbStart"/>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Від</string>
     </property>
    </widget>
   </item>
   <item row="0" column="6">
    <widget class="QPushButton" name="bnStop">
     <property name="icon">
      <iconset resource="../../../../AuLib/picture.qrc">
       <normaloff>:/butons/pict/calendar_48x48.png</normaloff>:/butons/pict/calendar_48x48.png</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QDateTimeEdit" name="sbStop"/>
   </item>
   <item row="0" column="4">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>До</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="8">
    <widget class="QTableWidget" name="twReportAgr">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>31</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="bnStart">
     <property name="icon">
      <iconset resource="../../../../AuLib/picture.qrc">
       <normaloff>:/butons/pict/calendar_48x48.png</normaloff>:/butons/pict/calendar_48x48.png</iconset>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Відобразити за</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QComboBox" name="cbSelZm">
     <item>
      <property name="text">
       <string>Поточна зміна</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Попердня зміна</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Поточна+Попередея</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Інший діапазон</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Останніх 10 записів</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../../AuLib/picture.qrc"/>
  <include location="../auscada.qrc"/>
 </resources>
 <connections/>
</ui>
